test/load/allocation/grpc/runAllocation.sh

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+# The number of times you want allocation test to be run
+TESTRUNSCOUNT=${TESTRUNSCOUNT:-3}
+NAMESPACE=${NAMESPACE:-default}
+EXTERNAL_IP=$(kubectl get services agones-allocator -n agones-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+KEY_FILE=${KEY_FILE:-client.key}
```

Comment:
You can get all using the following

```
kubectl get secret allocator-client.default -n ${NAMESPACE} -ojsonpath="{.data.tls\.crt}" | base64 -d > client.crt
kubectl get secret allocator-client.default -n ${NAMESPACE} -ojsonpath="{.data.tls\.key}" | base64 -d > client.key
kubectl get secret allocator-tls -n agones-system -ojsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
```

Reply:
Should these instructions be part of: https://agones.dev/site/docs/installation/install-agones/helm/#reserved-allocator-load-balancer-ip ?

Then this test can point to that documentation?

Reply:
Thanks for the suggestion. Updated the runAllocation.go

Reply:
Mark, I missed your comment. Are you OK to also have these extracted as part of runAllocation.sh ?

Reply:
I don't mind either way. It would be good to document this handy trick somewhere though at the very least.

Reply:
+1